zaza/openstack/charm_tests/manila/tests.py

```diff
@@ -85,6 +86,31 @@ def test_manila_api(self):
     def _list_shares(self):
         return self.manila_client.shares.list()
 
+    @tenacity.retry(
+        retry=tenacity.retry_if_result(lambda ret: ret is not None),
+        wait=tenacity.wait_fixed(120),
+        stop=tenacity.stop_after_attempt(2))
+    def _retry_check_commands_on_units(self, cmds, units):
+        return generic_utils.check_commands_on_units(cmds, units)
```

Comment:
As this is only being used once, it might be more logical to put it inline in the code using a `with` statement instead?

e.g.

```python
for attempt in tenacity.Retrying(...):
    with attempt:
        ret = generic_utils.check_commands_on_units(cmds, units)
```

Reply:
Thanks